application/values.yaml

```diff
@@ -287,6 +287,25 @@ route:
     termination: edge
     insecureEdgeTerminationPolicy: Redirect
 
+#Values for SecretProviderClass CR
+secretProviderClass:
+  enabled: true
+  name: mongo-host
+  provider: vault
+  parameters:
+    vaultAddress: "http://vault:8200"
+    roleName: "testing-role"
+    objects: |
+      - objectName: "MONGO_HOST"
+        secretPath: "testing/data/mongoDb"
+        secretKey: "MONGO_HOST"
+  secretObjects:
+    - data:
+      - key: MONGO_HOST
+        objectName: host
+      secretName: secret-mongo-host
+      type: Opaque
```

Comment:
@sabkat is this is required just to specify one secret? how will it look like if I have 20 secrets?

Reply:
No sir, for multiple secrets can also be used.